components/MashCards/ContactCard.spec.tsx

```diff
@@ -12,4 +13,30 @@ describe('ContactCard', () => {
     expect(screen.getByText(mockedMashReferral.requestedSupport as string));
     expect(screen.getByText('Work on'));
   });
+  it('renders the high priority banner when the referral is high priority and it is a red RAG rating', () => {
+    const priorityMockReferral = mashReferralFactory.build({
+      referrer: 'Police - red',
+    });
+    render(<ContactCard mashReferral={priorityMockReferral} />);
+    expect(screen.getByText('High priority'));
+    expect(
+      screen.getByText(`${priorityMockReferral.clients[0]}`, { exact: false })
+    );
+    expect(screen.getByText(priorityMockReferral.referrer as string));
+    expect(screen.getByText(priorityMockReferral.requestedSupport as string));
+    expect(screen.getByText('Work on'));
+  });
+  it('renders the high priority banner when the referral is high priority and it is a amber RAG rating', () => {
+    const priorityMockReferral = mashReferralFactory.build({
+      referrer: 'Police - amber',
+    });
+    render(<ContactCard mashReferral={priorityMockReferral} />);
+    expect(screen.getByText('High priority'));
+    expect(
+      screen.getByText(`${priorityMockReferral.clients[0]}`, { exact: false })
+    );
+    expect(screen.getByText(priorityMockReferral.referrer as string));
+    expect(screen.getByText(priorityMockReferral.requestedSupport as string));
+    expect(screen.getByText('Work on'));
+  });
```

Comment:
Similar comment to the one added for `ScreeningCard` component but this has a clearer test description, nice! 👍